bus-mapping/src/operation.rs

```diff
@@ -661,10 +774,21 @@ impl<T: Op> Ord for Operation<T> {
 
 impl<T: Op> Operation<T> {
     /// Create a new Operation from an `op` with a `rwc`
-    pub fn new(rwc: RWCounter, op: T) -> Self {
+    pub fn new(rwc: RWCounter, rw: RW, op: T) -> Self {
+        Self {
+            rwc,
+            rw,
+            reversible: false,
```

Comment:
Neither associated const nor trait implementation would work I think, because some operation could be reversible or not depends on their situation (for example, nonce increment of tx never gets reversible, but it get reversible when internal create).

Reply:
And I guess then that an Enum for that might be to overcomplicate it.